Integrations/integration-Okta.yml

```diff
@@ -2,7 +2,6 @@ commonfields:
   id: okta
   version: -1
 name: okta
-system: true
 display: okta
```

Comment:
i am adding this automatically when populating via content update.
but you will need to add fromversion in that case